test/acceptance/cli-test.js

```diff
@@ -1461,6 +1471,99 @@ describe('ember-template-lint executable', function () {
       });
     });
 
+    describe('with --format options', function () {
+      it('should be able to load relative printer', async function () {
+        project.setConfig({
+          rules: {
+            'no-bare-strings': true,
+            'no-html-comments': true,
+          },
+          pending: [
+            {
+              moduleId: 'app/templates/application',
+              only: ['no-html-comments'],
+            },
+          ],
+        });
+        project.write({
+          app: {
+            templates: {
+              'application.hbs':
+                '<h2>Here too!!</h2><div>Bare strings are bad...</div><!-- bad html comment! -->',
+            },
+          },
+          'custom-printer.js': `
+            class CustomPrinter {
+              constructor(options = {}) {
+                this.options = options;
+                this.console = options.console || console;
+              }
+
+              print(results) {
+                this.console.log(\`errors: \${results.errorCount}\`);
+                this.console.log(\`warnings: \${results.warningCount}\`);
+                this.console.log(\`fixable: \${(results.fixableErrorCount + results.fixableWarningCount)}\`);
+              }
+            }
+
+            module.exports = CustomPrinter;
+          `,
+        });
+
+        let result = await run(['.', '--format', './custom-printer.js']);
+
+        expect(result.stdout).toMatchInlineSnapshot(`
+          "errors: 2
+          warnings: 1
+          fixable: 0"
+        `);
+        expect(result.stderr).toBeFalsy();
+      });
+
+      it('should be able to load printer from node_modules', async function () {
+        debugger;
+        project.setConfig({
+          rules: {
+            'no-bare-strings': true,
+            'no-html-comments': true,
+          },
+          pending: [
+            {
+              moduleId: 'app/templates/application',
+              only: ['no-html-comments'],
+            },
+          ],
+        });
+        project.write({
+          app: {
+            templates: {
+              'application.hbs':
+                '<h2>Here too!!</h2><div>Bare strings are bad...</div><!-- bad html comment! -->',
+            },
+          },
+        });
+
+        let nodeModulesDir = path.join(project.baseDir, 'node_modules');
+        let formatterDir = path.join(nodeModulesDir, 'ember-template-lint-formatter-test');
+
+        await copy(
+          path.resolve(__dirname, '../fixtures/ember-template-lint-formatter-test'),
+          formatterDir
+        );
```

Comment:
Hmm, this is odd. Maybe just write it in the `project.write` above?

Reply:
Used `fs.symlinkSync` instead.